components/match2/wikis/rocketleague/match_summary.lua

```diff
@@ -15,6 +15,7 @@ local Json = require('Module:Json')
 local Abbreviation = require('Module:Abbreviation')
 local String = require('Module:StringUtils')
 local Flags = require('Module:Flags')
+local Opponent = require('Module:Opponent')
```

Comment:
```suggestion
local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
```

this makes it so that when using the dev stuff it automatically also uses opponent dev module